cli/cli.ts

```diff
@@ -1906,6 +1906,11 @@ ${gcards.map(gcard => `[${gcard.name}](${gcard.url})`).join(',\n')}
             if (game.subtitle) targetStrings[`{id:game-subtitle}${game.subtitle}`] = game.subtitle;
         }
 
+        const kioskGames = targetConfig?.kiosk?.games;
+        for (const game of (kioskGames ?? [])) {
+            if (game.name) targetStrings[`{id:game-name}${game.name}`] = game.name;
+        }
```

Comment:
> @jwunderl I updated cli for crowdin strings. Can you check it?

Yes that looks correct way to upload, though I'd add in the game.description field too probably https://github.com/microsoft/pxt-arcade/pull/6122/files#diff-7af2d83d1e7bf6ab0775d6df17f48e5c94698f41efd19dd1f7ac205871c17ca6R377

```suggestion
        for (const game of (kioskGames ?? [])) {
            if (game.name) targetStrings[`{id:game-name}${game.name}`] = game.name;
            if (game.description)  targetStrings[`{id:game-description}${game.description}`] = game.description;
        }
```

and in case explanation on what the id: portion is for, here's how they look in crowdin ui
![image](https://github.com/microsoft/pxt/assets/5615930/c06591fc-83ab-44f4-8046-43ee2dfbeebc)